packages/nc-gui/components/smartsheet/Gallery.vue

```diff
@@ -353,50 +390,53 @@ watch(
 <style scoped>
 .nc-gallery-container {
   grid-auto-rows: 1fr;
-  grid-template-columns: repeat(auto-fit, minmax(230px, 1fr));
+  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+}
+
+.ant-card-body {
+  @apply !p-0;
 }
 
 :deep(.slick-dots li button) {
   background-color: black;
 }
 
 .ant-carousel.gallery-carousel :deep(.slick-dots) {
-  position: relative;
+  @apply !w-auto;
```

Comment:
All css should be written in class util format. e.g. `position: absolute` -> `absolute`, `height: auto` -> `h-auto` ... 

In this way, something like below is expected:

```
@apply !w-auto absolute h-auto bottom-[3px];
```